lib/components/tabs.vue

```diff
@@ -146,28 +142,28 @@
              * Move to next tab
              */
             nextTab() {
-                this.setTab(this.currentTab, false, 1);
+                this.setTab(this.currentTab + 1, false, 1);
             },
 
             /**
              * Move to previous tab
              */
             previousTab() {
-                this.setTab(this.currentTab, false, -1);
+                this.setTab(this.currentTab - 1, false, -1);
             },
 
             /**
              * Set active tab on the tabs collection and the child 'tab' component
              */
-            setTab(index, force, offset) {
-                offset = offset || 0;
+            setTab(index, force, direction) {
+                direction = this.sign(direction || 0);
 
                 // Prevent setting same tab!
-                if (!force && (index + offset) === this.currentTab) {
+                if (!force && index === this.currentTab) {
```

Comment:
the variable offset doesn't exist anymore.

If the tab is already active, we can save some processing power.  But maybe it might be safer to always run the code below, just to make sure only one tab is ever active at a time.